src/app/loadout-builder/generated-sets/utils.ts

```diff
@@ -189,8 +189,9 @@ export function filterGeneratedSets(
     matchedSets = sets.filter((set) => canAllModsBeUsed(set, lockedSeasonalMods));
 
     console.info(
-      `Filtered out ${setsBeforeFilter -
-        matchedSets.length} sets based on seasonal mod requirements`
+      `Filtered out ${
+        setsBeforeFilter - matchedSets.length
```

Comment:
really like the way that template literals are handled now! was sometimes frustrating when it would split things in cases like this.

Reply:
Yeah I’ve been really happy with how Prettier continues to improve. I wish I had it for some of the other languages I use!